### ISSUE TITLE
Large spacing between some elements in horizontal mode

### ISSUE THREAD

**Issue by zany130** [flightlessmango/MangoHud#1109]
**Describe the bug**
Large spacing between some elements in horizontal mode

**List relevant hardware/software information**
- Linux Distribution Garuda Linux
- MangoHud version 0.6.9.1.r124.gf667002-1
- GPU 6700XT

**To Reproduce**
1. Use the provided mangohud config
2. See error

**Expected behavior**
Everything fits nicely without a lot of spacing
**Screenshots**
https://cdn.discordapp.com/attachments/641749996001296427/1148464514552045639/Screenshot_14.png
**Additional context**
[MangoHud.txt](https://github.com/flightlessmango/MangoHud/files/12518607/MangoHud.txt)

**Comment by gort818** [comment]
Does `hud_compact=1` make it any better for you?

**Comment by zany130** [comment]
yeah that what I been using for now but that makes some items too compact and there is some left over space I was looking for a way to basically configure the compactness of `hud_compact`

**Comment by gort818** [comment]
Does this patch work for you? Set the `10` to whatever you like
[hud_compact.txt](https://github.com/flightlessmango/MangoHud/files/12908158/hud_compact.txt)

I can make a new param so we can adjust that value from config file

**Comment by zany130** [comment]
Doesn't seem to make any difference on spacing of the elements tried different values 

`ImGui::SetCursorPosX(pos.x  + 5);`

**Comment by gort818** [comment]
hmm seems to work for me
Here is + 10
![x=10](https://github.com/flightlessmango/MangoHud/assets/9098121/ea1c41b5-9e14-4c61-8cc4-39eeb31836cb)
Here is + 50
![x=50](https://github.com/flightlessmango/MangoHud/assets/9098121/eab7298b-1655-4757-afa0-ce637c863a51)

**Comment by zany130** [comment]
Just tried again by removing the cloned folder and replacing the 10 with 50 no difference

here is what I ran
```
git clone --recurse-submodules https://github.com/flightlessmango/MangoHud.git
cd MangoHud
wget https://github.com/flightlessmango/MangoHud/files/12908158/hud_compact.txt
micro hud_compact.txt
git apply hud_compact.txt
./build.sh build
build/release/usr/bin/mangohud vkcube
```

before
![image](https://github.com/flightlessmango/MangoHud/assets/25832539/0d17d299-2ff4-40de-8df8-828bbee44b92)

after

![image](https://github.com/flightlessmango/MangoHud/assets/25832539/18531247-2907-41cb-8524-08f5a9ac4f64)

**Comment by flightlessmango** [comment]
by design all values has a buffer of 4 characters so the HUD doesn't move every time a value goes from 9 to 10 to 100 to 1000
